### ISSUE TITLE
[CAIA Intake]: Authenticated experience, profile hub, content review

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#66937]
### What does your team need support for?

- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [X] Making revisions to one or more existing VA.gov pages
- [ ] Something else

### Describe what your team needs support with

We got some feedback on the CAIA approved descriptive content for 3 sections in the profile hub during our midpoint review, and have been asked to make changes. We'd would like to get changes reviewed and approved ASAP. We're conducting user interviews/usability testing using production next week. It would be great to get veteran feedback on CAIA-approved content.

[Here's the current state of the hub page](https://www.sketch.com/s/b41a687f-8ab8-4be1-b189-692e5818cffd/a/27Yp2GA), which reflects the content we had reviewed in office hours a few weeks ago. The content we're referring to is under the heading in each card.

### Personal information
[Sketch file of personal information page, for context](https://www.sketch.com/s/ba254d92-3c3d-4eba-825d-d7f5bda35565/a/4ajJxKZ)

Current hub page description:
>Review your legal name, date of birth, and disability rating. And manage your preferred name and gender identity.

**Proposed update:** We'll remove the word "And" at the start of the second sentence. 
>Review your legal name, date of birth, and disability rating. Manage your preferred name and gender identity.

### Contact information
[Sketch file of contact information page, for context](https://www.sketch.com/s/59857eb5-d9f9-4145-99d3-d9a1de2d0655/a/098x48j)

Current hub page description:
>Manage your addresses, phone numbers, and contact email address.

**Proposed update:** We'll distinguish between addresses, and remove the word "contact" before email address.
>Manage your home and mailing address, phone numbers, and email address.

### Account security:
[Sketch file of account security page, for context](https://www.sketch.com/s/ebd4596f-0707-46cb-941e-247a808725cc/a/OmJwWpv)

Current hub page description: 
>Review your sign-in information and account setup.

**Proposed update:** We'll clarify exactly what a person can review in the "Account setup" section of the page.
>Review your sign-in information and the steps you’ve taken to secure your account.

### Will this new product be released incrementally?

- [X] Yes, we'll be launching incrementally
- [ ] No, we won't

### When do you expect to launch your product to 100% of users?

TBD

### Supporting artifacts (if any)

I don't think this is necessary for this review.

### Will this be going through the collaboration cycle?

- [X] Yes, it will be going through the collaboration cycle
- [ ] No, it will not

### When does this work need to be done?

ASAP, we're running usability testing starting 10/11 and need to make these content updates in production. And there is a federal holiday on 10/9.

### Do you plan to bring this to an upcoming content office hours session?

- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [X] Yes, health content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

### Who's on your team?

Team name: See ticket name
OCTO-DE product owner: Samara Strauss
Product manager: Travis Cahill
Designer: Liz Lantz
FE engineer: Adam Whitlock
Product/team Slack channel: #accountexp-authexp
...

### Do you have an accessibility champion on your team?

Yes, we have a champion

### If you have an accessibility champion, what's their name?

Liz Lantz

**Comment by andaleliz** [comment]
Reviewed with Danielle in office hours today.  

**Personal info:** We decided to stick with the current sentence structure since that reflects the voice/tone in other parts of the site.

**Contact info:** We talked about how the feedback is a little conflicting because on one hand _not_ distinguishing between the addresses is ambiguous, but using "contact" before email address is too specific.   Danielle said I can make the call as the designer, and I'd like to go with:
> Manage your addresses, phone numbers, and the email address we'll use to contact you.

My reasoning is that I hypothesize addresses is clear, and we know there's been confusion with email addresses in the past and we want to make sure it's clear to people which email address they can update on the contact information page.

**Account security:** We'll update it as follows:

> Review your sign-in and account information.
